### DISCUSSION TITLE
role permissions [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2328] [Discord Threads]
> 🙋‍♂️ **AK**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### currently no. role permissions will be enhanced in the future  
#  
> 🙋‍♂️ **esteve**  
#### I would be interested in such a functionality too. My need is more about allowing views /edit on some tables only or on some records only.  
#  
> 🙋‍♂️ **esteve**  
#### Is it possible to restrict access for the editors to some records only ? E.g. the apple salesman shall only be able to view & edit the records with the “apple” attribute and the oranges salesman shall only be able to view & edit the records with the “orange” attributes ? Thanks a lot  
![image0](https://cdn.discordapp.com/attachments/974670180380704849/982212336356048926/image0.jpg)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974670180380704849)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
